viewer/db.js

```diff
@@ -25,6 +25,7 @@ const internals = {
   arkimeNodeStatsCache: new Map(),
   shortcutsCache: new Map(),
   shortcutsCacheTS: new Map(),
+  queryExtraIndicesRegex: new Array(),
```

Comment:
This `internals` array holds regular expression(s) generated from the index pattern(s) specified in `queryExtraIndices`. These are used later to determine if an index matches one of the `queryExtraIndices` patterns, which are converted from wildcard to regex format using `ArkimeUtil.wildcardToRegexp` below.